ydb/docs/en/core/reference/ydb-cli/export-import/_includes/tools_restore.md

```diff
@@ -26,13 +26,17 @@ To import data to the table, use the [YQL `REPLACE` command](../../../../yql/ref
 
 `--restore-indexes VAL`: Enables/disables import of indexes, 1 (yes) or 0 (no), defaults to 1. If set to 0, the import won't either register secondary indexes in the data schema or populate them with data.
 
+`--restore-acl VAL`: Enables/disables import of access control lists (ACL) and owners for database objects, 1 (yes) or 0 (no), defaults to 1.
+
 `--dry-run`: Matching the data schemas in the database and file system without updating the database, 1 (yes) or 0 (no), defaults to 0. When enabled, the system checks that:
 
 - All tables in the file system are present in the database
 - These items are based on the same schema, both in the file system and in the database
 
 `--save-partial-result`: Save the partial import result. If disabled, an import error results in reverting to the database state before the import.
 
+`--import-data`: Use ImportData – a more efficient method to upload data. ImportData will throw an error if you try to upload data into an existing table that has secondary indexes or is in the process of building them. If you need to restore a table with secondary indexes, make sure it's not already present in the scheme. By default ImportData is disabled.
```

Comment:
This needs a link to some description of this ImportData and what it does.

Reply:
added a more specific description